client/src/helpers/APICalls/loginDemoUser.ts

```diff
@@ -0,0 +1,18 @@
+import { AuthApiData } from '../../interface/AuthApiData';
+import { FetchOptions } from '../../interface/FetchOptions';
+
+const loginDemoUser = async (): Promise<AuthApiData> => {
+  const fetchOptions: FetchOptions = {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({}),
```

Comment:
This can be removed

Reply:
Ethan,

I'm not sure if you saw my PR comment about the proxy error so I'll past it again here:
````
Proxy error: Could not proxy request /auth/demo from localhost:3000 to http://localhost:3001.
````
This error occurs when I click the 'Login As Demo User' button. 

Removing fetchOptions does not resolve this error. Please help me understand the problem. All other routes are working.

Reply:
Super odd fix - in the package.json the proxy should be http://localhost:3001**/** it's currently just http://localhost:3001

Reply:
Everything else looks fine to me & works after doing that fix in the proxy. Not a thorough fix but it gets it working - if we wanted to get a better fix in we would turn to the underlying webpack configuration doe the client dev server proxy settings.